.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "submodules/Atlus-Script-Tools"]
+	path = submodules/Atlus-Script-Tools
+	url = https://github.com/tge-was-taken/Atlus-Script-Tools.git
+	ignore = all
+[submodule "submodules/XV2-Tools"]
```

Comment:
We should think about changing the subdir for external dependencies to "external"